inst/docs/comparing-and-deploying/deploy.md

```diff
@@ -282,6 +283,50 @@ dL <- LinearMixedModelDeployment$new(p2)
 dL$deploy()
 ```
 
+##Deploying and Writing to non-default schemas in SQL Server
+
+As of the end of July 2017, there are some known issues with the packages `DBI`
+and `odbc`.  These issues get in the way of those writing to non-default schemas
+in SQL-Server. We hope to have the implementation of `DBI` and `odbc` fixed soon
+but we are waiting on the developers of those resepctive packages to fix the 
+issue. For now, we have created a work-around using the package `RODBC`.
+
+__Note__: `RODBC` is difficult to install on Mac OS.  This work-around is mainly for
+Windows users.  If Mac users can get`RODBC` installed on their machines, then the
+same code will work, but the installation will not be trivial.
+
+You may need to install RODBC using install.packages("RODBC"). Here is an 
+example of the work-around. 
+
+```r
+#First, create a table in SQL Server using a non-default schema. The example 
+#creates this table in the SAM database on localhost. You may also need to 
+#create a new schema(Cardiovascular) in SSMS for this specific example to work.
+#CREATE TABLE [Cardiovascular].[TestTable](
+#[a] [float] NULL,
+#[b] [float] NULL,
+#[c] [varchar](255) NULL)
+
+#create a connection to work with
+con <- odbcDriverConnect('driver={SQL Server};
+                              server=localhost;
+                              database=SAM;
+                              trusted_connection=true')
+
+#build a df to write to SQL Server
```

Comment:
Maybe make a note saying that the DF column names and types must match the SSMS table?